torch/csrc/jit/codegen/cuda/test/test_gpu3.cpp

```diff
@@ -6599,21 +6598,21 @@ TEST_F(NVFuserTest, FusionRepro2094_CUDA) {
   {
     auto tv0 = TensorViewBuilder()
                    .ndims(1)
-                   .shape({-1})
```

Comment:
This is not related to this PR, but @xwang233 reported that this line is causing compilation error on our nightly CI. I can not reproduce it locally, but this is my guessing fix.